third_party/hdf5.BUILD

```diff
@@ -0,0 +1,363 @@
+'''
+hdf5@1.12.2
+'''
+
```

Comment:
Let's add a comment pointing to where we got this from (I'm assuming it was tensorflow?). See here for an example: https://github.com/swift-nav/rules_swiftnav/blob/cb09e9ada2a509827143db3f192ca1f0bac30b9c/third_party/openssl.BUILD#L11.

Reply:
And our copyright notice.